internal/pkg/aws/ecr/ecr.go

```diff
@@ -167,9 +168,30 @@ func (s Service) DeleteImages(images []Image, repoName string) error {
 	return nil
 }
 
+func (s Service) repoExist(repoName string) (bool, error) {
+	_, err := s.ecr.DescribeRepositories(&ecr.DescribeRepositoriesInput{
+		RepositoryNames: aws.StringSlice([]string{repoName}),
+	})
+	if err != nil {
+		aerr := err.(awserr.Error)
+		if aerr.Code() != "RepositoryNotFoundException" {
+			return false, err
+		}
+		return false, nil
```

Comment:
Is it safe to return a `nil` error here and say the repo doesn't exist? This seems like a general unhandled error scenario which makes it seem unsafe to assume the repo doesn't exist?

Reply:
:+1: